src/View/ThemeResourceLoader.php

```diff
@@ -90,6 +90,14 @@ public function getSet($set)
         return null;
     }
 
+    /**
+     * Get the base path of the application according to the resource loader
+     */
+    public function getBase(): string
+    {
+        return $this->base;
+    }
```

Comment:
Needed for `SSTemplateEngine::findTemplate()` and to simplify some of the tests.